### DISCUSSION TITLE
Can message be more than one lines

### DISCUSSION THREAD

**Discussion by bincdianshangcun** [davatorium/rofi#1842] [Q&A] [answered]
In script doc:
https://github.com/davatorium/rofi/blob/next/doc/rofi-script.5.markdown#passing-mode-options

Message can be set as:
`echo -en "\0message\x1fHello message\n"`

How do I set the message to be multiple lines? I have tried below:
`"\0message\x1fLine 1\nLine 2\n"`
`"\0message\x1fLine 1\\\nLine 2\n"`
`"\0message\x1fLine 1\x1fLine 2\n"`

Thanks

**Comment by DaveDavenport** [comment, marked as answer]
```bash
#!/usr/bin/env bash

if [[ "$@" = "quit" ]]
then
    exit 0
fi

# Override the previously set delim.
# We only want to do this on first call of script.
if [[ $ROFI_RETV = 0 ]]
then
    echo -en "\x00delim\x1f\\x1\n"
fi
echo -en "\x00message\x1fmy line1\nmyline2\nmy line3\x1"
echo -en "\x00prompt\x1fChange prompt\x1"
for a in {1..10}
do
    echo -en "$a\x1"
done
echo -en "newline\ntest\x1"
echo -en "quit"
```

**Reply by bincdianshangcun** [reply]
It worked, thanks @DaveDavenport
